storage-bigtable/init-bigtable.sh

```diff
@@ -16,7 +16,7 @@ if [[ -n $BIGTABLE_EMULATOR_HOST ]]; then
   cbt+=(-project emulator)
 fi
 
-for table in blocks tx tx-by-addr; do
+for table in blocks entries tx tx-by-addr; do
```

Comment:
Presumably, someone has to manually create this extra table. Will that be covered under the "docs in another PR" ?

Reply:
> Will that be covered under the "docs in another PR" ?

Yep! That's right. In addition to updating docs in https://github.com/solana-labs/solana-bigtable, Joe and I are planning to do some comms with the folks already running bigtables about how to update their instances.